FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/FlowcryptApiRepository.kt

```diff
@@ -46,14 +51,21 @@ class FlowcryptApiRepository : ApiRepository {
 
   override suspend fun getDomainOrgRules(
     context: Context,
-    loginModel: LoginModel
+    loginModel: LoginModel,
+    fesUrl: String?
   ): Result<DomainOrgRulesResponse> =
     withContext(Dispatchers.IO) {
       val apiService = ApiHelper.getInstance(context).retrofit.create(ApiService::class.java)
       getResult(
         context = context,
         expectedResultClass = DomainOrgRulesResponse::class.java
-      ) { apiService.getDomainOrgRules(loginModel) }
+      ) {
+        if (fesUrl != null) {
+          apiService.getDomainOrgRules(fesUrl = fesUrl)
+        } else {
+          apiService.getDomainOrgRulesInternal(body = loginModel)
```

Comment:
The distinction here is not clear enough. `getOrgRulesFromFes` and `getOrgRulesFromFlowCryptComBackend` would be clearer.